lib/Pool.js

```diff
@@ -373,6 +381,7 @@ class Pool extends EventEmitter {
   _scheduleEvictorRun() {
     // Start eviction if set
     if (this._config.evictionRunIntervalMillis > 0) {
+      // @ts-ignore
       this._scheduledEviction = setTimeout(() => {
```

Comment:
Some things I just marked for TS ignore, just to avoid more complex type annotations.